pom.xml

```diff
@@ -204,22 +209,20 @@
   <profiles>
     <!-- Build distribution statically linked to BoringSSL -->
     <profile>
-      <id>boringssl</id>
+      <id>boringssl-static</id>
```

Comment:
I assume you're adding "-static" with the idea that we might eventually add a profile to build dynamically against Boring/Libre/Open?

Given that the artifact name is going to be `netty-tcnative-boringssl`, we should probably be consistent with the use of "-static" in the name for both the artifact and the profile name. WDYT?